docs/src/tutorials/Getting started/an_introduction_to_julia.jl

```diff
@@ -91,12 +91,12 @@ nothing #hide
 π
 
 # !!! tip
-#     To make π (and most other greek letters), type `\pi` and then press
+#     To make π (and most other Greek letters), type `\pi` and then press
 #     `[TAB]`.
 
 typeof(π)
 
-# Athough if we do math with irrational numbers, they get converted to
+# Although if we do math with irrational numbers, they get converted to
```

Comment:
I am not sure this sentence should start with "Although". I would rather use "However".

Reply:
```suggestion
# However, if we do math with irrational numbers, they get converted to
```